pkg/windows/windows.go

```diff
@@ -250,7 +250,7 @@ func (vm *windows) Run(cmd string, psCmd bool) (string, error) {
 	if err != nil {
 		// Hack to not print the error log for "sc.exe qc" returning 1060 for non existent services
 		// and not print error when the command takes too long to return after removing HNS networks.
-		if !(strings.HasPrefix(cmd, serviceQueryCmd) && strings.HasSuffix(err.Error(), serviceNotFound)) &&
+		if !(strings.Contains(cmd, serviceQueryCmd) && strings.Contains(out, serviceNotFound)) &&
```

Comment:
I've checked on all ways to run this (powershell, powershell with cmd, cmd, cmd with powershell), and the output always contained the `Failed 1060` error. I think it would be superfluous to check for  `status 1060` in the error message.